spec/fixtures/config_with_envs.yml

```diff
@@ -0,0 +1,8 @@
+# File is for testing out if ERB can be used within config.yml
+
+debug: true
+queue_name: <%= 'erb_' %>test:pmpy_queue
```

Comment:
Why do we need to set up the queue here? This gem should not use the queue

Reply:
once again it is used for testing:
spec/swift_ingest/ingestor_spec.rb
29:      sample_file = 'spec/fixtures/config_with_envs.yml'

I will update its contest to indicate that this file used for testing only.